addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetection.java

```diff
@@ -38,20 +41,13 @@
 public class PresenceDetection implements IPRequestReceivedCallback {
     public static final double NOT_REACHABLE = -1;
     NetworkUtils networkUtils = new NetworkUtils();
-
-    public enum PingMethod {
-        NONE,
-        JAVA_PING,
-        SYSTEM_PING
-    }
-
     private Logger logger = LoggerFactory.getLogger(PresenceDetection.class);
 
     /// Configuration variables
     private boolean useDHCPsniffing = false;
-    private boolean useARPping = false;
+    private ArpPingUtilEnum arpPingMethod = null;
```

Comment:
Leave out the null's, it will also be null if you don't specify

Reply:
Hm, for a new developer it might be easier this way actually, especially if it's another one with c++ background. Because with null an enumeration in java can actually express one more state, which is being used here and not obvious at a first glance.

Reply:
Okay